WordPress/src/androidTest/monkeys/playstore-screenshots.py

```diff
@@ -21,88 +21,51 @@ def action_login(device, serialno):
     device.press('KEYCODE_DPAD_DOWN')
     time.sleep(1)
 
-def action_open_drawer(device, serialno):
-    print("open drawer")
+def action_open_my_sites(device, serialno):
+    print("open my sites")
+    device.press('KEYCODE_TAB')
     for i in range(5):
-        device.press('KEYCODE_DPAD_UP')
         device.press('KEYCODE_DPAD_LEFT')
-    device.press('KEYCODE_ENTER')
-    # Wait for the animation to finish
-    time.sleep(1)
-
-# Menu drawer should be opened when calling this
-def action_open_notifications(device, serialno):
-    print("open notifications")
-    for i in range(5):
-        device.press('KEYCODE_DPAD_UP')
-    device.press('KEYCODE_DPAD_DOWN')
-    device.press('KEYCODE_DPAD_DOWN')
-    device.press('KEYCODE_ENTER')
-    # lose focus
-    time.sleep(1)
-    device.press('KEYCODE_DPAD_DOWN')
     # Wait for gravatars to load
-    time.sleep(10)
+    time.sleep(2)
 
-# Menu drawer should be opened when calling this
-def action_open_reader_posts_i_like(device, serialno):
+def action_open_reader_freshlypressed(device, serialno):
     print("open reader")
     # Open the reader
-    for i in range(3):
-        device.press('KEYCODE_DPAD_UP')
-    device.press('KEYCODE_DPAD_DOWN')
-    device.press('KEYCODE_ENTER')
```

Comment:
Ah, sorry I didn't catch that. Only have my rooted S3, wasn't testing on Lollipop.